Index: app/src/main/java/com/euphonyio/orderwith/data/DBUtil.kt
===================================================================
diff --git a/app/src/main/java/com/euphonyio/orderwith/data/DBUtil.kt b/app/src/main/java/com/euphonyio/orderwith/data/DBUtil.kt
--- a/app/src/main/java/com/euphonyio/orderwith/data/DBUtil.kt	
+++ b/app/src/main/java/com/euphonyio/orderwith/data/DBUtil.kt	
@@ -6,6 +6,7 @@
 import com.euphonyio.orderwith.data.dto.Menu
 import com.euphonyio.orderwith.data.dto.OrderMenu
 import com.euphonyio.orderwith.data.dto.OrderMenuItem
+import java.net.URL
 import java.util.*
 
 class DBUtil(context: Context) {
@@ -20,7 +21,7 @@
     }
 
     //  < Menu >
-    // 1-1. Get all menu (id, title, description, cost)
+    // 1-1. Get all menu (id, name, description, image, cost)
     fun getAllMenu(): List<Menu> {
         return db.menuDao().getAll()
     }
@@ -37,10 +38,10 @@
 
 
     // 2. add Menu
-    fun addMenu(title: String, description: String, image: String, cost: Int) {
+    fun addMenu(name: String, description: String, image: String, cost: Int) {
         val dao = db.menuDao()
         val id = dao.getLastId() + 1
-        val menu = Menu(id, title, description, image, cost)
+        val menu = Menu(id, name, description, image, cost)
         dao.insertMenu(menu)
     }
 
Index: app/src/main/java/com/euphonyio/orderwith/data/dto/OrderMenu.kt
===================================================================
diff --git a/app/src/main/java/com/euphonyio/orderwith/data/dto/OrderMenu.kt b/app/src/main/java/com/euphonyio/orderwith/data/dto/OrderMenu.kt
--- a/app/src/main/java/com/euphonyio/orderwith/data/dto/OrderMenu.kt	
+++ b/app/src/main/java/com/euphonyio/orderwith/data/dto/OrderMenu.kt	
@@ -13,6 +13,7 @@
  *  @author phj0407
  */
 @Entity(
+
     foreignKeys = [
         ForeignKey(
             entity = Order::class,
Index: app/src/main/java/com/euphonyio/orderwith/data/dao/MenuDao.kt
===================================================================
diff --git a/app/src/main/java/com/euphonyio/orderwith/data/dao/MenuDao.kt b/app/src/main/java/com/euphonyio/orderwith/data/dao/MenuDao.kt
--- a/app/src/main/java/com/euphonyio/orderwith/data/dao/MenuDao.kt	
+++ b/app/src/main/java/com/euphonyio/orderwith/data/dao/MenuDao.kt	
@@ -34,7 +34,10 @@
     fun insertMenu(menu: Menu) : Long
 
     @Delete
-    fun delete(menus: Menu)
+    fun delete(menu: Menu)
+
+    @Delete
+    fun deleteById(menuId:Int)
 
     @Query("DELETE FROM Menu")
     fun deleteAll()
Index: app/src/main/java/com/euphonyio/orderwith/data/dao/OrderDao.kt
===================================================================
diff --git a/app/src/main/java/com/euphonyio/orderwith/data/dao/OrderDao.kt b/app/src/main/java/com/euphonyio/orderwith/data/dao/OrderDao.kt
--- a/app/src/main/java/com/euphonyio/orderwith/data/dao/OrderDao.kt	
+++ b/app/src/main/java/com/euphonyio/orderwith/data/dao/OrderDao.kt	
@@ -1,9 +1,7 @@
 package com.euphonyio.orderwith.data.dao
 
-import androidx.room.Dao
-import androidx.room.Delete
-import androidx.room.Insert
-import androidx.room.Query
+import androidx.room.*
+import androidx.room.OnConflictStrategy.ABORT
 import com.euphonyio.orderwith.data.dto.Order
 
 /**
Index: app/src/main/java/com/euphonyio/orderwith/data/dao/OrderMenuDao.kt
===================================================================
diff --git a/app/src/main/java/com/euphonyio/orderwith/data/dao/OrderMenuDao.kt b/app/src/main/java/com/euphonyio/orderwith/data/dao/OrderMenuDao.kt
--- a/app/src/main/java/com/euphonyio/orderwith/data/dao/OrderMenuDao.kt	
+++ b/app/src/main/java/com/euphonyio/orderwith/data/dao/OrderMenuDao.kt	
@@ -30,5 +30,5 @@
     fun insertAll(vararg orderMenus: OrderMenu)
 
     @Delete
-    fun delete(orderMenus: OrderMenu)
+    fun delete(orderMenu: OrderMenu)
 }
\ No newline at end of file
